### DISCUSSION TITLE
ipq807x切换到5.15内核后出现问题

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#113] [Bugs/问题反馈]
nss貌似有问题，如图所示（拿网友的图），望修复，感谢
![4bca15a040c26da822c31c8a22658639_162892468-be174b9d-6fbf-4d87-8885-089d6baab4d5](https://user-images.githubusercontent.com/89022914/163502084-52a11898-e7bf-492a-b8e3-30004c372b99.png)

**Comment by Boos4721** [comment]
CHECK PINED

**Reply by ghost** [reply]
O...K
